.github/workflows/lint-cpp.yml

```diff
@@ -0,0 +1,55 @@
+name: Lint C++ Files
+
+on:
+  pull_request:
+    branches:
+      - release/**  # Triggers on pull requests targeting release branches
+  push:
+    branches:
+      - release/**  # Triggers on pushes to release branches, including merges
+
+jobs:
+  lint-cpp:
+    runs-on: ubuntu-latest
```

Comment:
Juste faire attention parce que ça va pointer vers Ubuntu 24.04 LTS soo pourrais créer des compatibility issues down the line pi j'ai comme l'impression ce serais caliss tough a debug. Peut être le garder sur 22.04

Reply:
true, done